femmt/examples/basic_transformer_5_windings.py

```diff
@@ -0,0 +1,78 @@
+import femmt as fmt
```

Comment:
This whole file should follow the rules from other basic_xxx.py-files.
Otherwise, the automated tests by CI-workflow via github can not be performed.

Check out one of the existing files, there is a `if __name__ == '__main__'` function. Furthermore, the whole example is wrapped into a function. And also, the thermal simulation should be included into this example